gendall-storybook-gatsby/src/stories/checkbox/Checkbox.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+
+import { CheckboxComponent } from "./Checkbox"
+
+export default {
+  title: "Filters/Checkbox",
```

Comment:
I think the checkbox component is a lower level component and should just be in a base controls category.